packages/wonder-blocks-core/src/util/types.js

```diff
@@ -16,26 +16,31 @@ type roles =
     | "alertdialog"
     | "application"
     | "article"
+    | "articles"
     | "banner"
     | "button"
+    | "cell"
     | "checkbox"
     | "columnheader"
     | "combobox"
-    | "command" // abstract role
+    | "command"
```

Comment:
Could you please leave this line as it was before? I think it's important to state that this is an abstract role and it shouldn't be included in any custom implementation.

`| "command" // abstract role`

> Abstract roles are used for the ontology. Authors MUST NOT use abstract roles in content.

More about it here: https://www.w3.org/TR/wai-aria/#abstract_roles
https://www.w3.org/TR/wai-aria/#isAbstract